### PR TITLE
refactor evaluation policies

### PR DESCRIPTION
**Pull request by thiagodks**
refactor evaluation policies and removed some policies

### CODE REVIEW

**Review by ncsilvaa — APPROVED** [review]

File: irec/evaluation_policies/fixed_interaction.py [hunk 1]
```diff
@@ -11,6 +10,17 @@
 
 
 class FixedInteraction(EvaluationPolicy):
+
+    """FixedInteraction
+    
+    Each user will be selected for 𝑇 times. Thus, the system will perform 𝑇 × |𝑈 | iterations,
+    where |𝑈 | is the number of distinct users available for the evaluation. The number 𝑇 is 
+    predefined by the researcher in a con- figuration file. Each user is randomly selected and
+    each action will not be per- formed more than once for him/her.
```
Comment (heitor57) [inline irec/evaluation_policies/fixed_interaction.py:19]: Fix these typos

File: irec/evaluation_policies/limited_interaction.py [hunk 2]
```diff
@@ -1,11 +1,22 @@
...
 
 class LimitedInteraction(EvaluationPolicy):
+
+    """LimitedInteraction
+
+    In this evaluation policy, the system will perform new actions until it
+    reaches all items registered in the user's history. The idea is to do an
+    exhaustive experiment to observe which algorithm takes the longest to reach
+    all the items previously evaluated by each user. Each user is randomly selected and
+    each action will not be per- formed more than once for him/her.
```
Comment (heitor57) [inline irec/evaluation_policies/limited_interaction.py:16]: Fix these typos

File: tests/integration/test_integration.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        ),
+    ],
+)
+def test_integration(models, datasets, metrics):
+
+    os.system(f"cd app && python3 run_agent_best.py --agents {models} --dataset_loaders '{datasets}' --evaluation_policy FixedInteraction --forced_run > out.txt")
+    # print("\nOIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII\n")
+    # os.system("ls")
+    assert 2 == 3, "ok"
+    # pass
```
Comment (heitor57) [inline tests/integration/test_integration.py:20]: This test should be in the irec-cmdline repository

**Review by heitor57 — CHANGES_REQUESTED** [review]
Just make a few changes and it's ready to merge

File: irec/evaluation_policies/fixed_interaction.py (hunk 1 above)
Comment (thiagodks) [reply irec/evaluation_policies/fixed_interaction.py]: revised

File: irec/evaluation_policies/limited_interaction.py (hunk 2 above)
Comment (thiagodks) [reply irec/evaluation_policies/limited_interaction.py]: revised

File: tests/integration/test_integration.py (hunk 3 above)
Comment (thiagodks) [reply tests/integration/test_integration.py]: removed

**Review by heitor57 — APPROVED** [review]

**Pull request merged**